demos-europe/demosplan-addon
# F t29878 disconnect addon data storage

**Ticket:** https://yaits.demos-deutschland.de/T29878

Description:
- We want to remove the dependency between DemosCore and addons
- DemosCoreEntities should not be used directly in demospipes neither maillane. Instead, interfaces provided by the middlelayer addons should be used.
- This PR is about linking the DemosCoreEntities to its corresponding Interfaces in the middlelayer Addon. 

How to review/test
- This PR is related to demospipes and maillane
- The 3 PRs should be merged to be able to test

Delete the checkbox if it doesn't apply/isn't necessary.
- [ x] Move the tickets on the board accordingly
